libs/docs/core/table/examples/table-fixed-example.component.html

```diff
@@ -0,0 +1,41 @@
+<div fd-table-fixed style="height: 300px">
```

Comment:
This actually breaks BEM. We use fd-table--fixed on the element that is not fd-table.
On my opinion, thats should be somehow resolved.

Reply:
We need to address this in styles (should have been part of https://github.com/SAP/fundamental-styles/pull/4214), and I guess may require substantial refactoring so not going to worry about this now.